components/brave_shields/common/brave_shields_panel.mojom

```diff
@@ -61,6 +63,13 @@ struct SiteSettings {
   bool is_noscript_enabled;
 };
 
+struct Rect {
```

Comment:
How about using `gfx.mojom.Rect` instead?

Reply:
the mojom Rect struct seems to left,top,right,bottom - I created a custom one to store width/height/x/y — perhaps i shouldn't call it Rect then as it can be confusing?

https://source.chromium.org/chromium/chromium/src/+/main:ash/components/arc/mojom/gfx.mojom;l=15?q=gfx.mojom

Reply:
We have proper one here - https://source.chromium.org/chromium/chromium/src/+/main:ui/gfx/geometry/mojom/geometry.mojom

Reply:
done